src/views/UserManagement.vue

```diff
@@ -0,0 +1,470 @@
+          <button class="button is-outlined is-text is-small is-loading document-loading">
+            Loading
+          </button>
+        </div>
+      </div>
+      <div v-if="addShown"
+           class="modal is-active">
+        <div class="modal-background">
+          <div class="modal-card is-size-7">
+            <header class="modal-card-head">
+              <h5 class="modal-card-title is-size-6 mb-0">
+                New user
+              </h5>
+              <button class="close-modal delete"
+                      aria-label="close"
+                      @click="addShown = false"/>
+            </header>
+            <section class="modal-card-body is-size-6 has-text-centered">
+              <div class="control content is-small mb-2">
+                Contact name
+              <input v-model="newUser.contact_name"
+                    class="input is-small"
+                    type="text"
+                    placeholder="Contact name">
+              </div>
+              <div class="control content is-small mb-2">
+                Email
+              <input v-model="newUser.email"
+                    class="input is-small"
+                    type="text"
+                    placeholder="Email">
+              </div>
+              <div class="control content is-small mb-2">
+                Mobile
+              <input v-model="newUser.mobile"
+                    class="input is-small"
+                    type="text"
+                    placeholder="Mobile">
+              </div>
+              <div class="control content is-small mb-2">
+                  Access Level
+                  <div class="select is-small is-fullwidth">
+                    <select v-model="newUser.acl"
+                            class="select is-small is-fullwidth"
+                            title="Select access level"
+                            :loading="isLoading">
+                            <option disabled value="">
+                              {{ newUser.acl? 'Select access level': 'Select access level' }}
+                            </option>
+                      <option v-for="accessLevel in accessLevels"
+                              :key="accessLevel.title"
+                              :value="accessLevel.value">
+                        {{ accessLevel.title }}
+                      </option>
+                    </select>
+                  </div>
+              </div>
+              <div class="control content is-small mb-2">
+                Organization
+                <div class="select is-small is-fullwidth">
+                  <select v-model="newUser.org_id"
+                          class="select is-small is-fullwidth"
+                          title="Select organization"
+                          :loading="isLoading">
+                          <option disabled value="">
+                            {{ newUser.org_id? 'Select organization': 'Select organization' }}
```

Comment:
Both options are the same, so we can just write `Select organization`

```suggestion
                            Select organization
```

Reply:
Done